### ISSUE TITLE
This project is great! But why the performance is still better on native python?

### ISSUE THREAD

**Issue by rafi16jan** [dmitrii-eremin/python-lua#5]
Hello, I am originally a Python enthusiast but after some point my clients can't keep up with Python's performance. So I was at an adventure for searching transpilers/translator or a even a language that have similar syntax (because that's what make python great, our team have a great consulting performance even sometimes we code in front of our clients, but not computing performance surely). And now we actually have already found a working transpiler called Rapydscript (python to JS), but still we want more. Lua is now one of the most stable language to be ran on WebAssembly. And then I search for python to Lua transpiler/translator and found your project which is awesome. But on our simple benchmarks Lua is not that faster compared to Python 3 even sometimes slower. And that's even the luajit, the native lua interpreter is 3x slower. Do you have any insights on this?

**Comment by greysondn** [comment]
Note that this isn't meant to make Lua *run* faster. It's meant to make it possible to program in Python for things that only accept Lua.

Lua is commonly used as an embedded language in video games and various engines. The syntax is a bit hard and object orientation in Lua is a bit esoteric. Having a language with more canonical syntax and semantics that can be translated to Lua with nominal overhead loss is an extremely good thing.

TL;DR - because you thought the point was that Lua was faster, when it's not. And that's just how it is.
